<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Logic Operations</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_result">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>100</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Результат</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_res_text">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>100</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Результат:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_operations">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Логическое И</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Логическое ИЛИ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Логическое НЕ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Эквивалентность</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Импликация</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Исключающее ИЛИ</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_op1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>False</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>True</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBox_op2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>False</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>True</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_op1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Переменная 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_op2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>80</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Переменная 2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Операции</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>467</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
